CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 
 * Add heading option to panel component ([PR #1741](https://github.com/alphagov/govuk_publishing_components/pull/1741)) MINOR
 * Force contents list title to always be Contents or regional equivalent ([PR #1734](https://github.com/alphagov/govuk_publishing_components/pull/1734)) BREAKING
+* BREAKING: Remove ability to pass Govspeak unsanitized HTML ([PR #1632](https://github.com/alphagov/govuk_publishing_components/pull/1632)) BREAKING
```

Comment:
The established convention is to mark breaking changes with `**BREAKING**` at the start. I don't think this line needs it at the end as well. If you look at the latest version the change on line 13 has been updated to match this.